content/en/glossary/terms/finding.md

```diff
@@ -0,0 +1,8 @@
+---
+title: finding
+core_product:
+  - security
+  - cspm
+---
+
+A finding is the primary primitive for a rule evaluation against a resource. Every time a resource is evaluated against a rule, a finding is generated with a Pass or Fail status.
```

Comment:
I know this content already exists and you're just moving it, but I don't like the capitalization of Pass or Fail. I would prefer bolding, perhaps? "A finding is generated with a **pass** or **fail** status." Just my preference, though, feel free to disregard.